docs/tutorials/02-channel-upgrades/05-upgrade-channel.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Upgrade channel
+sidebar_label: Upgrade channel
+sidebar_position: 5
+slug: /channel-upgrades/upgrade-channel
+---
+
+# Upgrade the ICS 20 transfer channel
+
+## Start the relayer
+
+We start the relayer:
+
+```bash
+hermes --config config.toml start
+```
+
+## Initiate the upgrade
+
+The [initiation of the upgrade process is authority-gated](https://ibc.cosmos.network/main/ibc/channel-upgrades#governance-gating-on-chanupgradeinit), so in this example the message to initiate the upgrade will execute when a governance proposal passes. The contents of the governance proposal are:
+
+```json title=proposal.json
+{
+  "title": "Channel upgrade init",
+  "summary": "Channel upgrade init",
+  "messages": [
+    {
+      "@type": "/ibc.core.channel.v1.MsgChannelUpgradeInit",
+      "signer": "cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn",
+      "port_id": "transfer",
+      "channel_id": "channel-0",
+      "fields": {
+        "ordering": "ORDER_UNORDERED",
+        "connection_hops": ["connection-0"],
+        "version": "{\"fee_version\":\"ics29-1\",\"app_version\":\"ics20-1\"}"
+      }
+    }
+  ],
+  "metadata": "AQ==",
+  "deposit": "100005stake"
+}
+```
+
+where `cosmos10d07y265gmmuvt4z0w9aw880jnsr700j6zn9kn` is the address of the governance module on `chain1`. The upgrade will modify the channel version to include the fee version.
+
+We submit the proposal:
+
+```bash
+simd tx gov submit-proposal ./proposal_upgrade_channel.json --from cosmos1vdy5fp0jy2l2ees870a7mls357v7uad6ufzcyz \
+--chain-id chain1 \
+--keyring-backend test \
+--home ../../gm/chain1 \
+--node http://localhost:27000
+```
+
+Now we vote for the proposal:
+
+```bash
+simd tx gov vote 1 yes \
+--from cosmos18phmkrpnn6gmpzscf6hnf5zpv06sygxc6f2v92 \
+--chain-id chain1 \
+--keyring-backend test \
+--home ../../gm/chain1 \
+--node http://localhost:27000
+```
+
+And we wait for the voting period to end. Once it ends we can check that the proposal has passed (i.e. the status has changed from `PROPOSAL_STATUS_VOTING_PERIOD` to `PROPOSAL_STATUS_PASSED`):
```

Comment:
We should specify how long the user needs to wait

Reply:
Added it.